frontend-gd/src/main/pages/log-in-page/google-login.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import ReactGoogleLogin from 'react-google-login';
+import { Redirect } from 'react-router-dom';
+
+const KEY = "525476460151-027bhu8spj34pr2gcm7o37vnd9r016ss.apps.googleusercontent.com"
```

Comment:
Could be wrong but I think we need to move our keys out of our files. With tutti I think we had all keys like locally saved and in a git ignore or something.

Reply:
Sounds good, this will probably fit under your auth stuff so do with it as you see fit.